COVID_HUNGARY_REPORT.wls

```diff
@@ -1028,234 +640,164 @@ PlotStyle->{
-{RGBColor[0.06666666666666667`,0.06666666666666667`,0.09803921568627451`], Thickness[0.0075]}},
-PlotLabel->"Mortality",
-PlotLegends->ageStructureLegend, AxesStyle->Directive[Gray,FontSize->20], LabelStyle->Directive[Bold, 20],
-Filling->{
-{7-> {{6}, RGBColor[0.06666666666666667`,0.06666666666666667`,0.09803921568627451`]}},
-{6-> {{5}, RGBColor[0.1764705882352941`,0.2823529411764706`,0.39215686274509803`]}},
-{5-> {{4}, RGBColor[0.8823529411764706`,0.2549019607843137`,0.07058823529411765`]}},
-{4-> {{3}, RGBColor[0.9372549019607843`,0.4627450980392157`,0.19215686274509802`]}},
-{3-> {{2}, RGBColor[0.9568627450980391`,0.7607843137254902`,0.3254901960784314`]}},
-{2-> {{1}, RGBColor[0.2549019607843137`,0.7176470588235294`,0.7058823529411764`]}},
-{1-> {Axis, RGBColor[0.615686274509804`,0.8196078431372549`,0.7333333333333333`]}}},
-Ticks->{{{0,"1. May"},{31+30+31,"1. August"},{31+30+31+31+30+31+30,"1. December"}}, 
-Automatic}]
-
-
-Export[StringJoin[generated,"elderly_hosp_2.pdf"],plt]
-
-
-plt=Plot[Evaluate[mortalityCumulative/.solElderlyFull],{t,0,250},
-PlotRange->{0,20000},
-PlotStyle->{
-{RGBColor[0.615686274509804`,0.8196078431372549`,0.7333333333333333`], Thickness[0.0075]},
-{RGBColor[0.2549019607843137`,0.7176470588235294`,0.7058823529411764`], Thickness[0.0075]},
-{RGBColor[0.9568627450980391`,0.7607843137254902`,0.3254901960784314`], Thickness[0.0075]},
-{RGBColor[0.9372549019607843`,0.4627450980392157`,0.19215686274509802`], Thickness[0.0075]},
-{RGBColor[0.8823529411764706`,0.2549019607843137`,0.07058823529411765`], Thickness[0.0075]},
-{RGBColor[0.1764705882352941`,0.2823529411764706`,0.39215686274509803`], Thickness[0.0075]},
-{RGBColor[0.06666666666666667`,0.06666666666666667`,0.09803921568627451`], Thickness[0.0075]}},
-PlotLabel->"Mortality",
-PlotLegends->ageStructureLegend, AxesStyle->Directive[Gray,FontSize->20], LabelStyle->Directive[Bold, 20],
-Filling->{
-{7-> {{6}, RGBColor[0.06666666666666667`,0.06666666666666667`,0.09803921568627451`]}},
-{6-> {{5}, RGBColor[0.1764705882352941`,0.2823529411764706`,0.39215686274509803`]}},
-{5-> {{4}, RGBColor[0.8823529411764706`,0.2549019607843137`,0.07058823529411765`]}},
-{4-> {{3}, RGBColor[0.9372549019607843`,0.4627450980392157`,0.19215686274509802`]}},
-{3-> {{2}, RGBColor[0.9568627450980391`,0.7607843137254902`,0.3254901960784314`]}},
-{2-> {{1}, RGBColor[0.2549019607843137`,0.7176470588235294`,0.7058823529411764`]}},
-{1-> {Axis, RGBColor[0.615686274509804`,0.8196078431372549`,0.7333333333333333`]}}},
-Ticks->{{{0,"1. May"},{31+30+31,"1. August"},{31+30+31+31+30+31+30,"1. December"}}, 
-Automatic}]
+PlotRangePadding->{{0, 0}, {0, Scaled[0.02]}}, Background->White,FrameTicks->{
+{0,60,120,180,240, 300, 360, 420},
+{{2000, "2000 "},{4000, "4,000 "},{6000, "6,000 "},{8000, "8,000 "}}}]
 
 
-Export[StringJoin[generated,"elderly_hosp_3.pdf"],plt]
+(* ::Subsubsection:: *)
+(*Calculation for determining the impact *)
```

Comment:
These sections "Calculation for determining the impact" could be simple text cells, instead of low-level headers.